### ISSUE TITLE
ModuleNotFoundError & Couldn't add tkdesigner to the PATH error

### ISSUE THREAD

**Issue by Kasi21** [ParthJadhav/Tkinter-Designer#319]
When trying to run the GUI, I get the following errors:
- ModuleNotFoundError: No module named 'PIL'
- RuntimeError: Couldn't add tkdesigner to the PATH.

Here are the solutions I've tried:
- Downloading the latest commit & rerunning the command to install requirements (all up to date)
- Running using the command 'python -m tkdesigner.cli $FILE_URL $FIGMA_TOKEN' (gives me the PIL error from before)

Could you please help me in resolving this?

Thank you in advance.

![image](https://user-images.githubusercontent.com/72465323/224558676-1058ed2d-68b5-4d68-8826-b4d8e8c2ba85.png)

**Comment by ok-coder1** [comment]
Have you added Python to the PATH?

**Comment by Kasi21** [comment]
Yes, Python is added to the PATH, and I'm running other Python scripts okay

**Comment by ok-coder1** [comment]
#169 Go here

**Comment by Kasi21** [comment]
Thanks, I tried that previously (I just tried it again, to be sure).

It doesn't resolve the issue.

**Comment by aellas** [comment]
Hey @Kasi21 
Have you tried to install pillow manually?
